deployment/terraform/coordinator.go

```diff
@@ -0,0 +1,125 @@
+package terraform
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	"github.com/mattermost/mattermost-load-test-ng/coordinator"
+	"github.com/mattermost/mattermost-load-test-ng/coordinator/agent"
+	"github.com/mattermost/mattermost-load-test-ng/coordinator/cluster"
+	"github.com/mattermost/mattermost-load-test-ng/coordinator/performance"
+	"github.com/mattermost/mattermost-load-test-ng/coordinator/performance/prometheus"
+	"github.com/mattermost/mattermost-load-test-ng/deployment/terraform/ssh"
+	"github.com/mattermost/mattermost-server/v5/mlog"
+)
+
+// Start starts the coordinator that is deployed.
+func (t *Terraform) StartCoordinator() error {
+	if err := t.preFlightCheck(); err != nil {
+		return err
+	}
+
+	output, err := t.getOutput()
+	if err != nil {
+		return err
+	}
+
+	if len(output.Agents.Value) == 0 {
+		return fmt.Errorf("there are no agent instances to run the coordinator")
+	}
+	ip := output.Agents.Value[0].PublicIP
+
+	loadtestConfig := t.generateLoadtestAgentConfig(output)
```

Comment:
Don't think we need this anymore since this config is injected by the `ltcoordinator` command.

Reply:
Yes, you spotted a good issue here. So, `ltcoordinator` uses it's own config file to seed agents. Instead of attaching this to `coordinator.Config` we should send this as a regular `config.json` file.

Reply:
Yes, problem is we actually use `loadtest.Config` in a couple of places in the cluster and agent packages. Might need more thinking. In the mean time I think we can safely remove this from here to keep the coordinator's config clean.